evm/chains/sharding/mainchain_handler/mainchain_handler.py

```diff
@@ -0,0 +1,78 @@
+import time
+
+import rlp
+
+from eth_utils import (
+    to_checksum_address,
+)
+
+from evm.chains.sharding.mainchain_handler.config import (
+    GASPRICE,
+    TX_GAS,
+)
+
+
+class MainchainHandler:
+
+    def __init__(self, web3_instance):
```

Comment:
*naming nitpick*

Can you rename the constructor variable from `web3_instance` to `w3` for consistency.

Reply:
yep no problem.